components/org.wso2.carbon.identity.application.authenticator.facebook/src/main/java/org/wso2/carbon/identity/application/authenticator/facebook/FacebookAuthenticator.java

```diff
@@ -366,13 +368,38 @@ protected void buildClaims(AuthenticationContext context, Map<String, Object> js
                 }
             }
 
+            if (StringUtils.isNotEmpty(claimDialectUri)) {
+                claimDialectUri = claimDialectUri.substring(0, claimDialectUri.length() - 1);
```

Comment:
you can directly call method and get dialect

Reply:
done